application/management/commands/load_csv.py

```diff
@@ -56,26 +57,56 @@ def handle(self, *args, **options):
                 elif activity_level == 'planned' or activity_level == 'Planned':
                     processed_space['operational_status'] = "Planned"
                 elif activity_level == 'inactive':
-                    processed_space['operational_status'] = "Closed"
-
+                    processed_space['operational_status'] = "Closed" 
+               
+                # Gets the affiliation ManyToMany option 
+                affiliation = space.pop('network_affiliation', None)
+                affiliation_obj = None
+                print(affiliation)
+                if affiliation:
+                    affiliation_obj = AffiliationOption.objects.filter(name=affiliation).first()
+                    print(affiliation_obj)
 
+                # Gets the governance ManyToMany option 
+                governance = space.pop('governance', None)
+                governance_obj = None
+                if governance:
+                    governance_obj = GovernanceOption.objects.filter(name=governance).first()
+                    
+                # Gets the ownership ManyToMany option 
+                ownership = space.pop('ownership', None)
+                ownership_obj = None
+                if ownership:
+                    ownership_obj = OwnershipOption.objects.filter(name=ownership).first()
+                
                 # Fields that share the name of where they are going
                 for field in Space._meta.get_fields():
                     if not field.name in processed_space:
-                        processed_space[field.name]=space.pop(field.name, None)
+                        # (PEDRO) To solve #85 remove the many to many fields
+                        if type(field).__name__ is not "ManyToManyField":
+                            processed_space[field.name]=space.pop(field.name, None)
                 space = {field:space[field] for field in space if space[field]}
                 processed_space['other_data'] = space
-                processed_spaces.append(processed_space)
-            spaces = [Space(**space) for space in processed_spaces]
-            for space in spaces:
+                #processed_spaces.append(processed_space)
+                
+                '''(Pedro) Removed the for loop, there may be a performance hit
+                    but I did it to add the ManyToMany relationships'''
+                new_space = Space(**processed_space)
                 try:
-                    space.clean_fields()
+                    new_space.clean_fields()
                 except ValidationError as v:
                     for field in v.error_dict:
-                        space.other_data[field] = space.__dict__[field]
-                        setattr(space, field, None)
+                        new_space.other_data[field] = new_space.__dict__[field]
+                        setattr(new_space, field, None)
                 try:
-                    space.save()
+                    new_space.save()
```

Comment:
I think you want to put the save after you adjust the ManyToMany fields, or else they won't persist.

Reply:
I though the same, but the space id field needs to be created first so the relationship can exist. It seems that the relationship persists but I'm gonna add an additional save() just in case.